doc/contributing/hw/README.md

```diff
@@ -0,0 +1,10 @@
+# Contributing to Hardware
+
+OpenTitan is a silicon project, so naturally there is a lot of aspects of hardware to contribute to.
```

Comment:
```suggestion
OpenTitan is an open-source silicon project, so naturally there are lots of aspects of the hardware design to contribute to.
```

Reply:
Thanks for suggesting these changes, I have added them to the PR.